File: FizzBuzz/fb.java
class Solution {
    public List<String> fizzBuzz(int n) {
        List<String> res = new ArrayList<String>();
        for(int i = 1;i<=n;i++){
            boolean a = (i%3==0);
            boolean b = (i%5==0);
            if(a &&!b) res.add("Fizz");
            if(!a&&b) res.add("Buzz");
            if(a&&b) res.add("FizzBuzz");
            if(!a&&!b) res.add(Integer.toString(i));
        }
        return res;
    }
}

File: FizzBuzz/fb.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        res = [str(i) for i in range(1,n+1)]
        for i in range(1,n+1):
            a = i%3==0
            b = i%5 == 0
            if a and not b:
                res[i-1] = "Fizz" 
            if not a and b:
                res[i-1] = "Buzz"
            if a and b:
                res[i-1] = "FizzBuzz"
        return res